hardware/asic/skywater/Makefile

```diff
@@ -50,23 +51,27 @@ ADD:   DEL
 
 $(DEFINE): ADD
 	echo "\`define $@ $($@)"| sed 's/=/ /g' >> temp/iob-soc-defs.vh
+
 # ASIC generation
-asic:  $(VSRC) $(VHDR) $(DEFINE)
+asic:  $(VSRC) $(VHDR) $(DEFINE) 
 	$(MKDIR) $(OPENLANE_DESIGNS)/system/src 
 	$(MKDIR) $(OPENLANE_DESIGNS)/system/macros/lef
 	$(MKDIR) $(OPENLANE_DESIGNS)/system/macros/gds
 	cp  $(wildcard sram/*.lef) $(OPENLANE_DESIGNS)/system/macros/lef 	
 	cp  $(wildcard sram/*.gds) $(OPENLANE_DESIGNS)/system/macros/gds 		
 	cp  $(wildcard temp/*.v) $(OPENLANE_DESIGNS)/system/src
 	cp  $(wildcard temp/*.vh) $(OPENLANE_DESIGNS)/system/src
+	cp  $(wildcard temp/boot.hex) $(OPENLANE_DESIGNS)/system/src
 	cp run_interactive.tcl $(OPENLANE_HOME)
 	cp config.tcl $(OPENLANE_DESIGNS)/system
 	cd $(OPENLANE_HOME);
 
```

Comment:
Don't lease empty lines in a Makefile recipe. And do not commit commented code lines.

Reply:
is there anything commented here ?

Reply:
There is an addition of one necessary line

Reply:
It should be right next to these ones. Why the empty line with no target and then an indented line?